### DISCUSSION TITLE
I waited 3 days and received nothing.

### DISCUSSION THREAD

**Discussion by entroxx** [community/community#140358] [GitHub Education] [answered]
### Hi everyone,

![image](https://github.com/user-attachments/assets/47ccc171-9ed4-4389-90e3-7d2b5c2cf3bb)
I got accepted. Why didn't I receive my perks?

**Comment by davevad93** [comment, marked as answer]
Hi @entroxx ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @entroxx was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by entroxx** [reply]
Alright. Thank you.

**Reply by davevad93** [reply]
You're welcome @entroxx 👍

**Comment by t1noo7** [comment]
waiting is happiness, young man : 3

They have also anticipated your waiting with the closing sentence:

`Congrats! Your request for @entroxx was approved and your benefits will be available soon.`

**Reply by entroxx** [reply]
Just got verified. Thank you!
